pkg/epp/util/env/env_test.go

```diff
@@ -69,6 +70,80 @@ func TestGetEnvFloat(t *testing.T) {
 	}
 }
 
+func TestGetEnvDuration(t *testing.T) {
```

Comment:
can you add green path tests also for int and float env vars?
just to cover that their parser func is actually working as expected :)

Reply:
oops. page didn't fully load.